solidity/contracts/PetalLock.sol

```diff
@@ -45,8 +45,9 @@ IEnsPublicResolver constant ensPublicResolver = IEnsPublicResolver(ENS_PUBLIC_RE
 
 // fuse combinations
 uint32 constant PARENT_FUSES_TO_BURN = CANNOT_UNWRAP + PARENT_CANNOT_CONTROL;
-uint32 constant FINAL_CHILD_FUSES_TO_BURN = CANNOT_UNWRAP + CANNOT_BURN_FUSES + CANNOT_SET_RESOLVER + CANNOT_CREATE_SUBDOMAIN + PARENT_CANNOT_CONTROL + CANNOT_APPROVE + CAN_EXTEND_EXPIRY;
+uint32 constant FINAL_CHILD_FUSES_TO_BURN = CANNOT_UNWRAP + CANNOT_BURN_FUSES + CANNOT_SET_RESOLVER + CANNOT_SET_TTL + CANNOT_CREATE_SUBDOMAIN + PARENT_CANNOT_CONTROL + CANNOT_APPROVE + CAN_EXTEND_EXPIRY;
```

Comment:
While technically the same, I think `&` makes it more clear that this is a bitmask.
```suggestion
uint32 constant FINAL_CHILD_FUSES_TO_BURN = CANNOT_UNWRAP & CANNOT_BURN_FUSES & CANNOT_SET_RESOLVER & CANNOT_SET_TTL & CANNOT_CREATE_SUBDOMAIN & PARENT_CANNOT_CONTROL & CANNOT_APPROVE & CAN_EXTEND_EXPIRY;
```